exporters/dsexporter/dsexporter.go

```diff
@@ -43,13 +43,14 @@ func main() {
 	exporter := NewCustomCollector()
 	reg.MustRegister(exporter)
 
+	fmt.Println("Server is listening on http://localhost:8443/metrics")
+
 	http.Handle("/metrics", promhttp.HandlerFor(
 		reg,
 		promhttp.HandlerOpts{
 			EnableOpenMetrics: true,
 			Registry:          reg,
 		},
 	))
-	fmt.Println("Server is listening on http://localhost:8090/metrics")
-	http.ListenAndServe(":8090", nil)
-}
```

Comment:
Missing a new line

Reply:
@avi-biton updating it now. I was troubleshooting the code and with https it gives me the merics for port 8443. Any idea which port should I be using here?

Reply:
You may define whichever port you want, 8443 seems like a good choice :)

Reply:
Once it will be defined and deployed, we should be able to see it in the `targets` in openshift

Reply:
can see the targets but it says scrape failed. Ports are defined correctly 
![image](https://github.com/redhat-appstudio/o11y/assets/113047440/8c9bdd38-5f47-437c-9249-9dd6dc7ce2f4)